lti_consumer/lti_consumer.py

```diff
@@ -925,15 +924,18 @@ def _get_context_for_template(self):
 
         # use bleach defaults. see https://github.com/jsocol/bleach/blob/master/bleach/__init__.py
         # ALLOWED_TAGS are
-        # ['a', 'abbr', 'acronym', 'b', 'blockquote', 'code', 'em', 'i', 'li', 'ol',  'strong', 'ul']
```

Comment:
Comments like this are hard to maintain, Instead, The URL above is enough and will provide updated information. 

I would suggest instead of hardcoding these tags and attributes, just have a direct URL that points to documentation or in GitHub ( whichever makes more sense).